Couple of things

ChrisPortman opened the pull request:
Hi Joshua,

Thanks for the module, makes perlbrew pretty straight forward!

Just a couple of things:
  - I had an issue where the installer set the init file up with setenv commands instead of exports for some reason.  Apparently it uses a $SHELL var to work out what shell is in use and then sets accordingly.  One of the commits forces bash and the install sorted itself out.

  - I created a new define for switch to separate it from the perl install process and to allow users to manage the switch separately in their profiles.  Should also help to support multiple perls per root.

  - Added a switch option to the perl define, to allow the disabling of the switch as part of the perl install.

Thanks,
Chris

jhoblitt commented on manifests/switch.pp:
```diff
@@ -0,0 +1,56 @@
+# == Define: perlbrew::switch
+#
+define perlbrew::switch(
+  $version,
```
This param is unvalidated.

jhoblitt commented on manifests/switch.pp:
```diff
@@ -0,0 +1,56 @@
+    '/usr/bin',
+  ]
+
+  if $version == 'off' {
```
Maybe `undef` but be better than off?

jhoblitt commented:
@ChrisPortman Could you take a stab at fixing the tests?

ChrisPortman commented:
Hi Joshua,
Sorry for the delay. I've been away of late.

I changed a few things:
- "The $version is validated as a string and defaults to 'system' which turns off perlbrew.
- I've updated the code/tests so that the non-future tests pass.  Although, I will admit, I have not increased the coverage, just made sure the existing tests pass.

Thanks,
Chris
